scripts/linters/codeowner_linter_test.py

```diff
@@ -0,0 +1,210 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/linters/codeowner_linter.py."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import multiprocessing
+import os
+
+from core.tests import test_utils
+import python_utils
+
+from . import codeowner_linter
+from . import pre_commit_linter
+
+NAME_SPACE = multiprocessing.Manager().Namespace()
+PROCESSES = multiprocessing.Manager().dict()
+NAME_SPACE.files = pre_commit_linter.FileCache()
+FILE_CACHE = NAME_SPACE.files
+
+LINTER_TESTS_DIR = os.path.join(os.getcwd(), 'core', 'tests', 'linter_tests')
+MOCK_CODEOWNER_FILEPATH = os.path.join(LINTER_TESTS_DIR, 'invalid_codeowners')
+
+CODEOWNER_IMPORTANT_PATHS = [
+    '/core/controllers/acl_decorators*.py',
```

Comment:
Do we need to kwwp this ins sync with CODEOWNER file? [I don't think so still if yes the add a comment heraeand in the CODEOWNER file asking users to keep this in sync.]

Reply:
No it is not necessary.

Reply:
no, we don't need to do any changes here if we change `CODEOWNER_IMPORTANT_PATHS`